config/hohgant/squashfs.py

```diff
@@ -0,0 +1,244 @@
+# Copyright 2016-2022 Swiss National Supercomputing Centre (CSCS/ETH Zurich)
+# ReFrame Project Developers. See the top-level LICENSE file for details.
+#
+# SPDX-License-Identifier: BSD-3-Clause
+#
+# ReFrame CSCS settings
+#
+import json
+import os
+
+import reframe.core.launchers.mpi as mpi
+import reframe.utility.osext as osext
+
+
+# @mpi.register_launcher('squashfs-run')
+# class MyLauncher(mpi.SrunLauncher):
+#     def run_command(self, job):
+#         return ' '.join(
+#             self.command(job) + self.options +
+#             [f'--uenv-mount-file={uenv_mount_file}',
+#              f'--uenv-mount-point={uenv_mount_point}',
+#             ]
+#             # ['squashfs-run', '$USER_ENV_IMAGE']
+#         )
```

Comment:
I think it's not very nice to merge files with commented out lines. But since the squashfs testing in work in progress we can leave it if it might be useful in the future.